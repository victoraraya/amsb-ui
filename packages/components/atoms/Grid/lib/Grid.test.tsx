import { render } from '@testing-library/react';
import '@testing-library/jest-dom';

import AirMilesGrid from './Grid';

it('should render successfully', () => {
    const { asFragment, getByText } = render(<AirMilesGrid>Hey!</AirMilesGrid>);

    const button = getByText('Hey!');

    expect(button).toBeInTheDocument();

    expect(asFragment()).toMatchSnapshot();
});
