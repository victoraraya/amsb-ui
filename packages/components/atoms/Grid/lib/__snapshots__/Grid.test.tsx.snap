// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render successfully 1`] = `
<DocumentFragment>
  .emotion-0 {
  box-sizing: border-box;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

<div
    class="MuiGrid-root emotion-0"
  >
    Hey!
  </div>
</DocumentFragment>
`;
